content/1_General/Input_Output.mdx

```diff
@@ -949,17 +1016,25 @@ public class Main {
 			pw = new PrintWriter(new FileWriter("paint.out"));
 		} catch(IOException e) {}
 	}
-	public static void main(String[] args) throws IOException {
 
-		int a = r.nextInt(), b = r.nextInt();
-		int c = r.nextInt(), d = r.nextInt();
+	public static void main(String[] args) throws IOException {
+		int a = r.nextInt();
+		int b = r.nextInt();
+		int c = r.nextInt();
+		int d = r.nextInt();
 
 		int[] cover = new int[100];
-		for (int i = a; i < b; i++) cover[i] = 1;
-		for (int i = c; i < d; i++) cover[i] = 1;
-		int ans = 0;
-		for (int i = 0; i < 100; i++) ans += cover[i];
+		for (int i = a; i < b; i++) {
+			cover[i] = 1;
+		}
+		for (int i = c; i < d; i++) {
+			cover[i] = 1;
+		}
```

Comment:
would it be better to use a bool array?

Reply:
i mean you're adding the numbers later on
doesn't matter that much tbh

Reply:
alr